index.rst

```diff
@@ -1,59 +1,367 @@
+The top-level ``HSC/defaults`` collection will include ``HSC/masks`` as well.
+
+.. _collections-reference-catalogs:
+
+Reference catalogs
+------------------
+
+External reference catalogs reformatted and sharded by DM code are written to ``refcats/<ticket>`` ``RUN`` collections, where ``<ticket>`` is the ticket on which the reformatting and sharding work was performed.
+After a reference catalog has been validated, its ``RUN`` is added to the overall ``refcats`` ``CHAINED`` collection.
+
+Different collections for different reference catalogs are not necessary, as the name of a reference catalog (e.g. ``ps1_pv3_3pi_20170110``) is used directly as its dataset type name (note that this was not the case in Gen2, where the reference catalog name was instead part of the data ID).
+
+.. _collections-skymap-definitions:
+
+SkyMap definitions
+------------------
+
+All skymaps must have a globally unique name in Gen3, which is used as part of the data ID for any dataset that is defined on tracts.
+The skymap definition datasets (i.e. ``lsst.skymap.BaseSkyMap`` subclass instances in Python) also include this globally unique name in their data IDs, and hence can also all go in a single ``skymaps`` collection.
+This is simply a ``RUN`` collection that holds skymap definition datasets directly.
+
+The existence of different skymap definition datasets for different coadd types (``goodSeeingCoadd_skyMap``, etc.) is a relic of Gen2 that will soon be removed entirely from Gen3; all skymap definition datasets will just use the ``skyMap`` dataset type.
+The new globally-unique skymap data ID names are both necessary and sufficient for uniqueness in Gen3.
+
+SkyMap registration is something we expect to be rare in Gen3 - *much* more rare than running ``makeSkyMap.py`` was in Gen2 - because we almost always use one of a few standard SkyMaps, and in Gen3 a SkyMap (a combination of a ``lsst.skymap.BaseSkyMap`` class *and* its configuration) can only be registered once.
+Discrete SkyMaps, which typically cover only a small part of the sky and are *conceptually* a bit more per-user, maybe less rare, but our data model currently does not treat these any differently, and until we can identify the patterns and use cases for creating new SkyMaps (even discrete ones), we propose that any new SkyMap registration in a shared repository be preceded by an RFC.
+
+.. _collections-shared-official-processing-outputs:
+
+Shared/official processing outputs
+----------------------------------
+
+Processing runs overseen by production operators should produce output collections of the form ``<instrument>/runs/<target>/<release>/<ticket>``, or ``runs/<target>/<release>/<ticket>`` in the (rare) case of processing that includes science data from multiple instruments and none of them can be considered the "primary" instrument.
```

Comment:
I think you and I (and other people at generalized Princeton/UW) are not really ever "production operators", at least in the construction Project - that's the title I'm using for (e.g.) the roles currently played by Monika and Eric at NCSA.  Beyond that, I'd defer to @MichelleGower on this question, as I picked up the terminology from her and mostly inferred its meaning (and the notion that there is a clear separation) from context in various conversations, rather than having a clear definition of it in my own head.

Reply:
That is interesting and not the answer I expected! Further clarification on this would be welcome.

Reply:
My assumption is that there's a production operator role.  Members of that role know when they are doing production runs vs doing personal science runs.  Monika and Eric are currently the folks at NCSA in that role (e.g., they are responsible for running the HSC RC2 pipelines).  (pre-) Operations will determine who those folks are for the data previews and data releases.  I'm pretty sure I haven't answered the question, but I'm not quite sure what the question is.  What was the answer you expected?

Reply:
That's closer to what I expected :) I guess I thought some project developers would share responsibility with data facility folks to keep new datasets ingested and/or processed. Thank you both, I recognize this is still being defined and precursor datasets are a bit of a gray area.